README.md

```diff
@@ -49,8 +59,9 @@ resource.gender
 ```
 
 ### [Documentation](https://dermatologist.github.io/fhiry/)
+
 ## Contributors
 
 * [Bell Eapen](https://nuchange.ca) | [![Twitter Follow](https://img.shields.io/twitter/follow/beapen?style=social)](https://twitter.com/beapen)
 * WIP, PR welcome, please see CONTRIBUTING.md
```

Comment:
Thank you @Mandalka! Feel free to add your name to the contributor's list, if you wish to.

Cheers
@dermatologist